ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -268,6 +269,17 @@ class FamilyReads extends React.PureComponent {
     locus: null,
   }
 
+  componentDidMount() {
+    const { variant, familyGuid, defaultSampleTypes } = this.props
+    if (defaultSampleTypes) {
+      this.setState({
+        openFamily: familyGuid,
+        sampleTypes: defaultSampleTypes,
+        locus: variant && getVariantLocus(variant, this.getProjectForFamily(familyGuid)),
```

Comment:
Add this to allow the IGV to be displayed without clicking the `ShowIgvButton`.

Reply:
IGV should only ever be show when a trigger is clicked. The behavior for IGV on this new page should not be substantively different than anywhere else- there is a trigger and there is an IGV panel

Reply:
We need to show the IGV when clicking on the `chrom/start/end/strand` columns in the outlier table. So the trigger is the `chrom/start/end/strand` fields. I think putting the trigger button provided by the `FamilyReads` into the outlier table is hard. And I don't think we should instantiate a `FamilyReads` instance for each row in the table to make the trigger buttons ready in rows.

Reply:
If you make the changes I propse to how you are instantiating `FamilyReads` this should not be needed and you will still be instantiating one family reads instance, you will just be able to call `showReads` when the link is clicked